pages/docs/user-docs/build-yum.md

```diff
@@ -1,16 +1,42 @@
 ---
-title: Yum Builds
+title: yum bootstrap module
 sidebar: user_docs
 permalink: build-yum
 folder: docs
 toc: false
 ---
 
+This module allows you to build a Red Hat/CentOS/Scientific Linux style container from a mirror URI. 
 
-The YUM base uses YUM on the host system to bootstrap the core operating system that exists within the container. This module is applicable for bootstrapping distributions like Red Hat, Centos, and Scientific Linux. When using the `yum` bootstrap module, several other keywords may also be necessary to define:
+{% include toc.html %}
+
+## Overview
+Use the `yum` module to specify a base for a CentOS-like container.  You must also specify the URI for the mirror you would like to use.  
+
+## Keywords
+```
+Bootstrap: yum
+```
+The **Bootstrap** keyword is always mandatory. It describes the bootstrap module to use.
+```
+OSVersion: 7
+```
+The **OSVersion** keyword is optional. It specifies the OS version you would like to use.  It is only required if you have specified a %{OSVERSION} variable in the `MirrorURL` keyword. 
+```
+MirrorURL: http://mirror.centos.org/centos-%{OSVERSION}/%{OSVERSION}/os/$basearch/ 
```

Comment:
just to clarify - OSVersion is optional here, but not for debootstrap?

Reply:
Yeah.  I think that is true.  `debootstrap` doesn't use the weird %{OSVersion} variable in the MirrorURL.